src/scry/protocol/symbol_information.cr

```diff
@@ -18,6 +18,7 @@ module Scry::Protocol
     Number      = 16
     Boolean     = 17
     Array       = 18
+    Annotation  = 19
```

Comment:
These values are defined by the LSP specification here: https://microsoft.github.io/language-server-protocol/specification#textDocument_documentSymbol

I do not believe there is an `Annotation` symbol type, we may have to use a different type or "bucket" it under an existing one.

Reply:
Looks like one of the Java LSP implementation maps their annotations to Properties: 
https://github.com/eclipse/eclipse.jdt.ls/blob/76934e4d7e9ff91d4000df8b4eaa3b942aef2311/org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/handlers/DocumentSymbolHandler.java#L246

Reply:
There was a PR to add it (microsoft/language-server-protocol#131) but this looks to have been closed in favour of further discussion around standardisation of symbols kinds (microsoft/language-server-protocol#344).

That discussion has already been running for ~2 years so its probably worth while using an interim mapping as you suggest. I'm not sure if the `Property` kind is the best fit semantically though. Given LSP springs from Microsoft's naming conventions, that term normally maps to members that expose access to fields on an object (see https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/properties, https://docs.microsoft.com/en-us/dotnet/fsharp/language-reference/members/properties). From a quick look through other language server implementations `SymbolKind::Class` also gets some use for this purpose, but there doesn't appear to be any form of consensus.

@bmulvihill as one of the existing contributors for scry, what's your preference?